docs/model_fields.rst

```diff
@@ -695,6 +695,45 @@ Or with attribute assignment to a model::
         any conflicts.
 
 
+.. _enum-fields:
+
+-----------
+Enum Fields
+-----------
+
+You can take advantage of MySQL's ``ENUM`` type through the ``EnumField``
+provided by django-mysql. Using ``ENUM`` provides the benefit of storing
+recurring strings in a very space efficient manner. MySQL will store the
+enumeration value in each row as an integer value and return results as
+the corresponding strings in queries.
+
+Specifying the range of values available in an EnumField must be specified
+in the `choices` argument as a tuple or list::
+
+    class BookCover(Model):
+        color = EnumField(choices=['red', 'green', 'blue'])
+
+Like Django's standard `choices` argument, the values may be specified as a
+tuple specifying the actual value and a human readable name. The actual value
+is what will be stored in the database, while the human readable value will be shown
+in Django Admin forms where applicable::
+
+    class Book(Model):
+        color = EnumField(choices=[
+          ('red', 'Bright Red'),
+          ('green', 'Vibrant Green'),
+          'blue',  # human readable name will be set to "blue"
+        ])
+
+
+.. NOTE::
+   It is possible to introduce new enumeration choices in a migration, as well as
+   edit the *human readable* names of current choices.
+
+   However, editing or removing existing choice values will result in corresponding
+   rows to be represented as empty strings.
```

Comment:
To be replaced with the empty string? Also make a note it's only possible to ``ALTER TABLE`` to truncate the data if MySQL Strict Mode is not on.